### DISCUSSION TITLE
poor text rendering at 16-bit color depth

### DISCUSSION THREAD

**Discussion by b-user** [tnodir/fort#152] [Q&A] [answered]
1. Set 16-bit color depth in OS
2. Try to exit Fort
3. See result

![16bit](https://github.com/tnodir/fort/assets/11581901/5210c54e-ec64-4204-9de7-2b05739e9e35)

Environment:
virtual box
windows 7 x86 (guest os)
16-bit color depth (in OS)

**Comment by tnodir** [comment, marked as answer]
The workaround for this bug is:
create `qt.conf` file in the Fort's installed directory (near of `FortFirewall.exe`) with the following content:
```
[Platforms]
WindowsArguments = dpiawareness=1,fontengine=freetype
```

**Comment by b-user** [comment]
It works, thanks.

[Platforms]
WindowsArguments = fontengine=freetype

![qt-freetype](https://github.com/tnodir/fort/assets/11581901/850a4be9-792e-4d59-82a6-b1d056d42550)
